py/src/braintrust/wrappers/test_anthropic.py

```diff
@@ -224,6 +243,25 @@ def test_anthropic_client_error(memory_logger):
     assert "404" in log["error"]
 
 
+def test_anthropic_messages_stream_errors(memory_logger):
+    assert not memory_logger.pop()
+
+    client = wrap_anthropic(_get_client())
+    msg_in = {"role": "user", "content": "what is 2+2? (just the number)"}
+
+    try:
+        with client.messages.stream(model=MODEL, max_tokens=300, messages=[msg_in]) as stream:
+            raise Exception("fake-error")
+    except Exception:
+        pass
```

Comment:
maybe we should also test that the exception `fake-error` gets raised out of the sdk back to the app?